dev/var_names_xportr.Rmd

```diff
@@ -34,12 +34,13 @@ The 6 main functions within `xportr` pacakge:
 * `xportr_write()`
 
 
-The demo will make use of a small `ADSL` data set that is apart of the  [`{admiral}`](https://pharmaverse.github.io/admiral/index.html) package. The script that generates this `ADSL` dataset can be created by using this command `admiral::use_ad_template("adsl")`.  
+The demo will make use of a small `ADSL` dataset available with the `xportr` package.
+The script that generates this `ADSL` dataset can be created by using the command: `admiral::use_ad_template("adsl")` _(note: `admiral` package needs to be installed).
```

Comment:
```suggestion
The script that generates this `ADSL` dataset can be created by using the command: `admiral::use_ad_template("adsl")` (note: `admiral` package needs to be installed).
```

maybe we should just drop this statement since the data is available to the user already.  ??

Reply:
Sounds good!

Reply:
Check: 909e6d4